Packs/OpsGenie/Integrations/OpsGenieV3/OpsGenieV3.yml

```diff
@@ -1500,6 +1516,29 @@ script:
     - contextPath: OpsGenie.Team.name
       description: Team name.
       type: String
+  - name: opsgenie-get-request
+    polling: true
```

Comment:
why should it be set to `polling` as well?

Reply:
any command which can return a scheduled result needs polling: true